sixtyfps_runtime/corelib/model.rs

```diff
@@ -285,13 +285,9 @@ impl<'a, T> Iterator for ModelIterator<'a, T> {
     type Item = T;
 
     fn next(&mut self) -> Option<Self::Item> {
-        if self.row < self.model.row_count() {
-            let row = self.row;
-            self.row += 1;
-            Some(self.model.row_data(row))
-        } else {
-            None
-        }
+        let row = self.row;
+        self.row += 1;
```

Comment:
From an overflow perspective it seems wrong to increase `self.row` - also given different behavior in debug vs. release. Maybe an overflow is not a realistic scenario?